code/game/gamemodes/dynamic/dynamic.dm

```diff
@@ -198,7 +202,12 @@ GLOBAL_VAR_INIT(dynamic_forced_threat_level, -1)
 
 /datum/game_mode/dynamic/send_intercept()
 	. = "<b><i>Central Command Status Summary</i></b><hr>"
-	switch(round(threat_level))
+	var/shown_threat
+	if(prob(FAKE_REPORT_CHANCE))
+		shown_threat = rand(1 to 100)
+	else
+		shown_threat = clamp(threat_level + rand(REPORT_NEG_DIVERGENCE, REPORT_POS_DIVERGENCE), 0, 100)
+	switch(round(shown_threat))
 		if(0 to 19)
 			if(!current_players[CURRENT_LIVING_ANTAGS].len)
 				. += "<b>Peaceful Waypoint</b></center><BR>"
```

Comment:
since the shown threat value for "dynamic greenshifts" can be pushed above 19 by the variance you added above, you might want to move this if statement outside of this switch statement.

Reply:
bonus points if you make it set the threat level to green when it runs

Reply:
no